code/modules/mob/living/simple_animal/bot/mulebot.dm

```diff
@@ -545,11 +541,19 @@
 
 /mob/living/simple_animal/bot/mulebot/Moved() //make sure we always use power after moving.
 	. = ..()
-	if(!cell)
-		return
-	cell.use(cell_move_power_usage)
+
+	cell?.use(cell_move_power_usage)
+
+	for(var/potential_target in loc.contents)
+		if(potential_target == load)
+			continue
+
+		if(ishuman(potential_target))
+			RunOver(potential_target)
+
 	if(cell.charge < cell_move_power_usage) //make sure we have enough power to move again, otherwise turn off.
```

Comment:
Sanity.

Reply:
What do you mean by sanity?

Reply:
you removed the check for it cell exists @ l548

Reply:
The check is now removed entirely. We now check cell?.use() before doing a bot_step and if this fails, the bot turns itself off and doesn't move.